atlasdb-ete-tests/src/test/java/com/palantir/atlasdb/ete/CassandraMultinodeEteTest.java

```diff
@@ -0,0 +1,228 @@
+/**
```

Comment:
Consider renaming it something like `CassandraMultinodePerformanceEteTest` and adding it to `CassandraMultinodeTestSuite`